vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -356,7 +394,7 @@ void AdsImpl::NotificationEventDismissed(
 void AdsImpl::NotificationEventTimedOut(
     const std::string& id,
     const NotificationInfo& notification) {
-  notifications_->Remove(id);
+  notifications_->Remove(id, false);
```

Comment:
When a notification times out it should also be dismissed as for desktop notifications time out after 2 minutes. This event does not occur for Android.